[2023-09-04T06:10:12.373+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2023-04-21T00:00:00+00:00 [queued]>
[2023-09-04T06:10:12.400+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2023-04-21T00:00:00+00:00 [queued]>
[2023-09-04T06:10:12.401+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:10:12.661+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2023-04-21 00:00:00+00:00
[2023-09-04T06:10:12.671+0000] {standard_task_runner.py:57} INFO - Started process 9316 to run task
[2023-09-04T06:10:12.685+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_data_from_table', 'scheduled__2023-04-21T00:00:00+00:00', '--job-id', '2143', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp88z2mzwf']
[2023-09-04T06:10:12.690+0000] {standard_task_runner.py:85} INFO - Job 2143: Subtask delete_data_from_table
[2023-09-04T06:10:12.895+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2023-04-21T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:10:13.144+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2023-04-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-21T00:00:00+00:00'
[2023-09-04T06:10:13.146+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2023-04-21' and dag_id = 'dag_with_postgres_operator_v04';
        
[2023-09-04T06:10:13.167+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:10:13.196+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:10:13.221+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2023-04-21' and dag_id = 'dag_with_postgres_operator_v04';
        , parameters: None
[2023-09-04T06:10:13.230+0000] {sql.py:427} INFO - Rows affected: 0
[2023-09-04T06:10:13.252+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_data_from_table, execution_date=20230421T000000, start_date=20230904T061012, end_date=20230904T061013
[2023-09-04T06:10:13.517+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:10:13.568+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
